app/docker/networks/network.helper.ts

```diff
@@ -0,0 +1,5 @@
+const systemNetworks: string[] = ['host', 'bridge', 'none'];
+
+export function isSystemNetwork(networkName: string): boolean {
```

Comment:
```suggestion
export function isSystemNetwork(networkName: string) {
```

see my comment above, ts knows it returns a boolean

Reply:
still here :)